### PR TITLE
Trigger WF runs on PR

### PR DESCRIPTION
**Pull request by Imipenem**
Run WF on push and pr

### CODE REVIEW

File: .github/workflows/create_cli_java.yml [hunk 1]
```diff
@@ -1,10 +1,11 @@
 name: Create cli-java Template
 
-on: 
+on:
   push:
     paths-ignore:
       - "tests/**"
       - "docs/**"
+  pull_request:
```
Comment (Zethson) [inline .github/workflows/create_cli_java.yml:8]: We should also ignore the paths here I guess. It brings CI runs down.

File: .github/workflows/create_cli_python.yml [hunk 2]
```diff
@@ -1,10 +1,11 @@
 name: Create cli-python Template
 
-on: 
+on:
   push:
     paths-ignore:
       - "tests/**"
       - "docs/**"
+  pull_request:
```
Comment (Zethson) [inline .github/workflows/create_cli_python.yml:8]: Apples here as well and to all other create workflows

**Review by Zethson — CHANGES_REQUESTED** [review]
I would add the ignore paths everywhere. Rest LGTM

**Comment by Imipenem** [comment]
I will fix this asap

**Comment by Imipenem** [comment]
@Zethson : There are issues with GitHub actions according to GitHub Status page, so I suggest wait for them to be resolved before merging!

**Review by Zethson — APPROVED** [review]
LGTM

**Pull request merged**